### PR TITLE
fix: close keyboard on action (AR-912)

### PR DESCRIPTION
**Pull request by trOnk12**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-912" title="AR-912" target="_blank">StoryAR-912</a>  Send text messages
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

On action click the keyboard was not closing, also when the input textfield was clicked the action was not closing

### Solutions

-Close actions on input field click
- On action click, reset the focus

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-472/1/) **failed**.

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-472/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3012607)(3012607)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3012607)(3012607)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-472/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3018674)(3018674)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3018674)(3018674)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposerInnerState.kt [hunk 1]
```diff
@@ -45,7 +45,7 @@ class MessageComposerInnerState(
 
     var attachmentOptionsDisplayed by mutableStateOf(false)
 
-    private fun toEnabled() {
+     private fun toEnabled() {
```
Comment (yamilmedina) [inline app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposerInnerState.kt:48]: There is a small/tiny miss alignment, the extra space at the start of the function 👀

**Review by yamilmedina — APPROVED** [review]
Just a small indentation issue, but everything looks good 👍 💯

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-472/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3019546)(3019546)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3019546)(3019546)-dev-release.apk)

**Pull request merged**